sormas-base/pom.xml

```diff
@@ -445,10 +445,11 @@
 				<scope>provided</scope>
 				<exclusions>
 					<!-- Exclude Payara modules here -->
-					<exclusion>
-						<groupId>net.minidev</groupId>
-						<artifactId>json-smart</artifactId>
-					</exclusion>
+					<!-- json-smart is a server module, but for unknown reason can't be loaded by the json-path domain lib -->
```

Comment:
1. Write: net.minidev:json-smart is a Payara module, but for unknown reason can't be loaded by the json-path domain lib
2. Remove the following commented out code